pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -3426,10 +3409,9 @@ func vmiToDomain(vmi *v1.VirtualMachineInstance, c *ConverterContext) *api.Domai
 
 func xmlToDomainSpec(data string) *api.DomainSpec {
 	newDomain := &api.DomainSpec{}
-	err := xml.Unmarshal([]byte(data), newDomain)
+	Expect(xml.Unmarshal([]byte(data), newDomain)).To(Succeed())
```

Comment:
This should be `ExpectWithOffset`. Or - IMO - better yet, return the error and perform the assertion on the caller.

You wouldn't even have to change the callers (assuming they invoke this func within the expectations) since `Expect` allows for functions with multiple return values (as you know).

Reply:
I used `ExpectWithOffset`. It was simpler.